federation/apis/federation/types.go

```diff
@@ -84,6 +84,11 @@ type ClusterStatus struct {
 	// Allocatable represents the total resources of a cluster that are available for scheduling.
 	Allocatable api.ResourceList `json:"allocatable,omitempty"`
 	ClusterMeta `json:",inline"`
+	// Zones is the list of avaliability zones in which the nodes of the cluster exist, e.g. 'us-east1-a'.
```

Comment:
You will need to make the same change to federation/apis/federation/v1alpha1/types.go